src/Widgets/SpaceWidget/SpaceWidgetClass.js

```diff
@@ -0,0 +1,23 @@
+import * as Scrivito from "scrivito";
+
+const SpaceWidget = Scrivito.provideWidgetClass("SpaceWidget", {
+  attributes: {
+    size: [
+      "enum",
+      {
+        values: [
```

Comment:
I suggest to use the short-cuts in the data-layer:

```suggestion
        values: ["xs", "s", "m", "l", "xl", "xxl"],
```